foundations/installations/text_editors.md

```diff
@@ -7,13 +7,86 @@ Rich text editors, such as Microsoft Word and Libre-Office Writer, are great for
 
 ### Code Editors
 
-You can think of code editors as specialized web development tools. They are highly customizable and offer many features that will make your life easier. There is nothing worse than spending 2 hours trying to figure out why your program isn't working only to realize that you missed a closing bracket. Plugins, syntax highlighting, auto-closing of brackets and braces, and linting are just a few of the benefits of using a code editor. There are many text editors out there to choose from, but we suggest starting with VSCode.
+You can think of code editors as specialized web development tools. They are highly customizable and offer many features that will make your life easier. There is nothing worse than spending 2 hours trying to figure out why your program isn't working only to realize that you missed a closing bracket. Plugins, syntax highlighting, auto-closing of brackets and braces, and linting are just a few of the benefits of using a code editor. There are many text editors out there to choose from, but we suggest starting with Visual Studio Code.
 
-**VSCode** is an excellent free code editor. It has outstanding add-on support and great Git integration. VSCode is the most popular code editor among Odin's students and moderators, so support is easy to find in the community. You can download the .deb file and install it from [here](https://code.visualstudio.com/).
+**Visual Studio Code**, or just VSCode as it's commonly referred to, is an excellent free code editor. It has outstanding add-on support and great Git integration. VSCode is the most popular code editor among Odin's students and moderators, so support is easy to find in the community.
 
 Which editor you use is generally a matter of preference, but for the purposes of this course, we are going to assume you're using VSCode, mainly because it's free, it's easy to use, and it works pretty much the same on every operating system. Keep in mind that means you will not able to get help if you are using a different text editor than VSCode for the curriculum.
 
-As a reminder, if you're using a **virtual machine**, you should install your text editor of choice **on your VM**. You're welcome to also install it on your host (i.e., your Windows main OS), but you'll want to be sure that you have this critical tool inside your VM.
+As a reminder, if you're using a **virtual machine**, you should install VSCode **on your VM**. You're welcome to also install it on your host (i.e., your Windows main OS), but you'll want to be sure that you have this critical tool inside your VM.
+
+### VSCode Installation
+
+Choose your Operating System:
+
+<details markdown="block">
+<summary class="dropDown-header">Linux</summary>
+
+#### Step 1: Download VSCode
+
+   - Open your **Terminal**
+   - Run the following command to download the latest **VSCode** `.deb` package
+
+~~~bash
+curl -L -o code-latest.deb 'https://code.visualstudio.com/sha/download?build=stable&os=linux-deb-x64'
+~~~
```

Comment:
Another small question I have.

 A quick google search says that we (TOP) have users first install curl in the [Installing Node.js lesson](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/installing-node-js#installing-nvm). I don't think it's ideal if users run the curl command and its not installed on their systems. 
 
 Do you think it's worth adding an instruction to install curl?

Reply:
We can change it to wget instead.